client/my-sites/stats/stats-email-top-row/top-card.jsx

```diff
@@ -10,7 +10,7 @@ const TopCardValue = ( { value, isLoading } ) => {
 	const isNumber = Number.isFinite( value );
 
 	if ( isLoading ) {
-		return <Spinner />;
+		return <Spinner baseClassName="calypso-spinner" />;
```

Comment:
The default `.spinner` class is hidden in WP-Admin, so we are using an alternative base class